lib/source.js

```diff
@@ -75,27 +76,24 @@ module.exports = class CovSource {
   // given a start column and end column in absolute offsets within
   // a source file (0 - EOF), returns the relative line column positions.
   offsetToOriginalRelative (sourceMap, startCol, endCol) {
-    const lines = this.lines.filter((line, i) => {
-      return startCol <= line.endCol && endCol >= line.startCol
-    })
+    const lines = sliceRange(this.lines, startCol, endCol)
     if (!lines.length) return {}
 
     const start = originalPositionTryBoth(
       sourceMap,
       lines[0].line,
       Math.max(0, startCol - lines[0].startCol)
     )
+    if (!(start && start.source)) {
```

Comment:
I rearranged the conditions so that it fails fast and doesn't call the expensive originalEndPositionFor when not necessary